data/json/bionics.json

```diff
@@ -1081,7 +1077,7 @@
     "type": "bionic",
     "name": { "str": "Integrated Toolset" },
     "description": "Surgically implanted in your hands and fingers is a set of tools - screwdriver, hammer, wrench, hacksaw, drill, welder, and heating elements.  You can use this in place of many tools when crafting.",
-    "occupied_bodyparts": [ [ "hand_l", 3 ], [ "hand_r", 3 ] ],
+    "occupied_bodyparts": [ [ "hand_l", 3 ], [ "hand_r", 1 ], [ "arm_l", 2 ], [ "arm_r", 2 ] ],
```

Comment:
```suggestion
    "occupied_bodyparts": [ [ "hand_l", 1 ], [ "hand_r", 1 ], [ "arm_l", 3 ], [ "arm_r", 3 ] ],
```

Reply:
Shift even more stuff from hands to arms?
It's a set of many tools with tips extending from your fingers/hands. Considering that there isn't that much space in hand to play with, 3;1 slots are justified.

Reply:
3 in one hand and 1 in the other?